test/metabase/feature_extraction/math_test.clj

```diff
@@ -0,0 +1,91 @@
+(ns metabase.feature-extraction.math-test
+  (:require [expectations :refer :all]
+            [metabase.feature-extraction.math :refer :all]))
+
+(expect
+  (approximately 5.5 0.1)
+  (transduce identity magnitude [1 2 3 4]))
+(expect
```

Comment:
spacing issues here

Reply:
bump

Reply:
This one is intentionally missing an empty line as it would have been just one expect if it weren't for the approximate which doesn't play nicely with vectors.

Reply:
can we add a comment saying as much? That way people know why it's done that way

Reply:
Sure. Also, I'm not super attached to this, just seemed like the right thing to do at the time.

Reply:
note added